node.js

```diff
@@ -419,8 +429,10 @@ module.exports = {
     var halfYearAgo = Date.now() - TIME_TO_UPDATE_CANIUSE
```

Comment:
Why you need `halfYearAgo` if you already calculated month?

Reply:
fixed